main.py

```diff
@@ -51,30 +74,27 @@ async def ping(message):
 
     @bot.command()
     async def register(message):
-      try:
-        spreadSheet.new_worksheet(f'{message.author.id}')
-        await message.channel.send(f'worksheet made!')
-          
-      except gspread.exceptions.APIError:
-        await message.channel.send(f'Spreadsheet by the name of <@{message.author.id}> already exists')
+#      try:
+      self.spreadsheet.new_worksheet(f'{message.author.id}')
+      await message.channel.send(f'worksheet made!')
+      
+#      except gspread.exceptions.APIError:
+#        await message.channel.send(f'Spreadsheet by the name of <@{message.author.id}> already exists')
 
     @bot.command()
     async def newHeader(message, *args):
-      if self.authorID_worksheet(message.author.id) == True:
+      if self.author_id_worksheet_not_found(message.author.id):
         await message.channel.send('you need to register a spreadsheet with ;register first')
       else:
         #gets all cell values of top row
-        rowValues = spreadSheet.get_row(1)
+        rowValues = self.spreadsheet.get_row(1)
         count = 0
         headers = ''
         
         #compiling arguments in message contents
-        for i in args:
-          if i != args[-1]:
-            headers += str(i) + ' '
-        headers += args[-1]
+        headers = " ".join(str(next_header) for next_header in args)
         
-        #find every second column so that I can fit datetine values for each cell (B, D, F, etc are all data columns while the other odds contain dates)
+        #find every second column so that I can fit datetime values for each cell (B, D, F, etc are all data columns while the other odds contain dates)
```

Comment:
Can you clarify this comment further? How do we know that each cell in columns B, D, F, H, ... will need their own corresponding timestamp? Won't it depend on what data is stored in the given cell, and therefore won't it depend on the header set by the user?
@megumin00 I'd appreciate a clarification of the requirements. If the bot is logging user messages, then I imagine the messages would be saved in column B with their timestamps in column A (right?). But what will be stored in column D that _also_ needs a timestamp separate from the timestamp given in column A at the same row number? I'd like to see an example.